packages/create-bison-app/template/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Default configuration to run your NextJs app locally:
```

Comment:
Is there a reason to use a `Dockerfile` for this? Could we just mount the entire `./` folder (excluding `.data` and likely a few other folders) in `docker-compose.yml` and run directly from there?